docs/topic-guides/blendable-web-apps/view-composing.md

```diff
@@ -48,6 +48,45 @@ Here, the content of the view is defined on the root level and the presentation
 
 The `declarative-shadow-dom` is used as the default composition that can be further modified or replaced in runtime using the Starcounter  app suite [Blending](https://github.com/Starcounter/Blending) and its feature - composition editor.
 
+## Subset and superset composition matching
+
+The key for the custom compositions is the main view. If there is more than one custom composition, the best matching composition is picked by its distance to the attached views:
+
+1. If there is an exact match of a custom composition to attached views, provide this composition
+1. Else, if there are superset custom compositions (compositions that include all of the attached views and more), provide the smallest superset
+1. Else, if there are subset custom compositions (compositions that include some of the attached views [*intersection*] and maybe some other views [*complement*]), provide the subset that has the largest intersection and smallest relative complement. For the attached views not included in the composition, use their default composition
+1. Else, use the default compositions
```

Comment:
I'm wondering if this would be clearer by writing in pseudo code instead of in plain text. What do you think?

Reply:
TBH I like it how it is, for me it's clean and simple to understand thanks to good description. 

I'm not sure in what exact form you would like to have it, can you write part of this pseudo code?